commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/model/ShipmentItem.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.order.web.internal.model;
+
+/**
+ * @author Alessio Antonio Rendina
+ */
+public class ShipmentItem {
+
+	public ShipmentItem(
+		long shipmentItemId, long orderId, String sku, int ordered, int shipped,
+		int toSend) {
+
+		_shipmentItemId = shipmentItemId;
+		_orderId = orderId;
+		_sku = sku;
+		_ordered = ordered;
+		_shipped = shipped;
+		_toSend = toSend;
+	}
+
+	public int getOrdered() {
+		return _ordered;
+	}
+
+	public long getOrderId() {
+		return _orderId;
+	}
+
+	public long getShipmentItemId() {
+		return _shipmentItemId;
+	}
+
+	public int getShipped() {
+		return _shipped;
+	}
+
+	public String getSku() {
+		return _sku;
+	}
+
+	public int getToSend() {
+		return _toSend;
+	}
+
+	private final int _ordered;
```

Comment:
Not sure if these names correspond with a pattern that's already established, but `_orderedCount` and `_shippedCount` would be better than `_ordered` and `_shipped`. As it is, looking at just the field names I would assume they're booleans -- adding `Count` as a suffix would make them a lot less ambiguous.